src/main.py

```diff
@@ -0,0 +1,73 @@
+class ChainSaw:
+    def __init__(self, name="DniproM", power=150, rotation_speed=250, category="Tool"):
+        self.__name = name
+        self.__power = power
+        self.__rotation_speed = rotation_speed
+        self.category = category  
+
+    # геттери / сеттери
+
+    @property
+    def name(self):
+        return self.__name
+
+    @name.setter
+    def name(self, name):
+        self.__name = name
+
+    @property
+    def power(self):
+        return self.__power
+    
+    @power.setter
+    def power(self, power):
+        self.__power = power
+
+    @property
+    def rotation_speed(self):
+        return self.__rotation_speed
+    
+    @rotation_speed.setter
+    def rotation_speed(self, rotation_speed):
+        self.__rotation_speed = rotation_speed
+
+
+    # перевизн методи
+    def __str__(self):
+        return f"chainsaw: {self.__name}, power: {self.__power} W, rotation speed: {self.__rotation_speed} rotations per second, category: {self.category}"
+
+    def __repr__(self):
+        return f"chainsaw (name='{self.__name}', power={self.__power}, rotation_speed={self.__rotation_speed}), category={self.category}"
```

Comment:
The format string in __repr__ should include category={self.category} inside the parentheses to match the actual class representation and improve readability

Reply:
Done